src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -309,6 +309,7 @@ Below is a list of available sub-commands:
 |:-----------------------------|:----------------------------------------------------------------------------|
 | `--kubeconfig` `<string>`    | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
 | `--kubecontext` `<string>`   | Kubeconfig context to use
+| `--in-cluster`              | Use the in-cluster configuration to connect to Kubernetes.
```

Comment:
What would be the behaviour of this flag when run as a stand-alone command and not as a pod?

Reply:
It will return error to user.